### ISSUE TITLE
Notice of moving from namecheap DNS management to Cloudflare

### ISSUE THREAD

**Issue by moonstash** [stakeordie/SecretWebsiteV2#104]
We will be migrating the DNS management from namecheap to cloudflare end of Q1. This change is needed so we are able to leverage cloudflare features that namecheap and render.com do not have, specifically this is useful for redirects and advanced features we need for the community api which lives on api.scrt.network. For some reason render.com doesn't play nice when domains dns are managed by cloudflare. This issue is merely a notice on this migration since the scrt.network website currently uses render.com

**Comment by dylanschultzie** [comment]
Yes please, this has been an ongoing issue, thus: https://github.com/stakeordie/SecretWebsiteV2/pull/102 

This helps *kind of*, but ultimately we don't know where all links are going to be coming from.

**Comment by mohammedpatla** [comment]
This would help us continue staying in this domain. Hopefully, Stakeordie can do this switch?

**Comment by moonstash** [comment]
> This would help us continue staying in this domain. Hopefully, Stakeordie can do this switch so all the other services work

It's certainly possible to migrate from render.com to cloudflare as seen here ----> https://scrtnetwork.pages.dev/
